rest/src/db/CatapultDb.js

```diff
@@ -491,8 +492,7 @@ class CatapultDb {
 					const newConditions = { _id: { $in: accountIds } };
 					// repeat the response with the found and sorted account ids, so that the result can be complete with all the mosaics
 					// Second query set pageIndex to 0;
-					options.pageNumber = 1;
-					return this.queryPagedDocuments(newConditions, [], {}, 'accounts', options)
+					return this.queryPagedDocuments(newConditions, [], {}, 'accounts', options, undefined, 0)
```

Comment:
instead of creating a new param, send new arguments as options. 

return this.queryPagedDocuments(newConditions, [], {}, 'accounts', {pageSize, pageNumber:1})

the original options object is not changed and we don't need to add an extra argument.

Reply:
Don't think it will work mate, the pageNumber in the pagination obj should return the original number in the query string

Reply:
Yesp, needs this too:

 fullAccountsPage.pagination.pageNumber = options.pageNumber;